spec/integration/mappers/overwrite_attributes_value_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'rom/memory'
+
+describe 'Mappers / Attributes value' do
+  let(:setup) { ROM.setup(:memory) }
+
+  before do
+    setup.relation(:users)
+  end
+
+  it 'allows to manipulate attribute value' do
+    class Test::UserMappe < ROM::Mapper
+      relation :users
+
+      attribute :id
+      attribute :name, from: :first_name do 'John' end
+      attribute :age do 9+9 end
+      attribute :weight do |t| t+15 end
```

Comment:
`attribute :foo { 1+1 }` fails because the braces bind much tighter than do/end -- ruby attempts to send the block to `:foo`, not to `attribute`

To work inline, you have to do `attribute(:foo) { 1 + 1 }`